workers/data_refinery_workers/processors/create_compendia.py

```diff
@@ -56,30 +56,23 @@ def log_state(message, job, start_time=False):
 
 def _prepare_input(job_context: Dict) -> Dict:
     start_time = log_state("prepare input", job_context["job"])
-    # We're going to use the smasher outside of the smasher.
-    # I'm not crazy about this yet. Maybe refactor later,
-    # but I need the data now.
-    job_context = smasher._prepare_files(job_context)
-    job_context = smasher._smash(job_context, how="outer")
 
-    if not 'final_frame' in job_context.keys():
-        logger.warn("Unable to prepare files for creating compendia.",
-            job_id=job_context['job'].id)
+    def log_failure(failure_reason: str) -> None:
```

Comment:
Just curious since you're declaring this inside `_prepare_input` - is this function not useful in any other context?